### PR TITLE
Update Readme.md file & added type definition

### PR DESCRIPTION
**Pull request by ANKDGIRI777**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
...
+# unleash-client
+
+This package provides the Unleash configuration used in Safe Security.
+
+## Motivation
+
+Currently in Safe Security, all the services that leverage Unleash functionality required to define the unleash communication mechanism which lead to boilerplate code in each of the services.
+Adhering to the DRY (Don't Repeat Yourself) principle, the common configuration has been extracted out as part of this repository and made available for consumption via NPM package.
+
+In order to solve this problem, the `Unleash` library used currently has been encapsulated in this package along with the desired defaults set up so that
```
Comment (prathap-safe) [inline README.md]: This paragraph is not needed as the above covers it.

File: README.md [hunk 2]
```diff
@@ -1 +1,49 @@
...
+The introduction of this package makes it very easy for clients to consume & leverage the unleash capabilities.
+
+In Javascript:
+
+```js
+const { getInstance } = require("unleash-client-safe-security");
+
+const unleash = getInstance({
+    unleashServerUrl: "https://YOUR-UNLEASH-URL",
+    baseUrl: "https://example.safescore.io",
```
Comment (prathap-safe) [inline README.md]: Use publicly available values!

**Review by prathap-safe — CHANGES_REQUESTED** [review]

**Comment by prathap-safe** [comment]
Update the PR title to match the actual changes made in the PR.

File: README.md (hunk 1 above)
Comment (ANKDGIRI777) [reply README.md]: Removed the paragraph

File: README.md (hunk 2 above)
Comment (ANKDGIRI777) [reply README.md]: Updated with publicly available values

**Review by aneeshrelan — APPROVED** [review]

**Pull request closed without merging**